docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-2-0,Logstash 5.2.0>>
+
+[[logstash-5-2-0]]
+=== Logstash 5.2.0 Release Notes
+
+* Added the ability to collect and report on cgroup information. This info is helpful when you're 
+  running Logstash in a container and the normal metrics like CPU and system load are still tracked at the 
+  machine level, not the container level ({lsissue}6252[Issue 6252]).
+* Persistent queue stats are now available in stats API when they are enabled. Metrics such as the number 
+  of events in the queue waiting to be processed and disk space info will be reported ({lsissue}6508[Issue 6508]).
+* Plugin outputs have been enhanced to report wall-clock execution time. This info is available in the 
+  `duration_in_millis` field in the node stats API response ({lsissue}6458[Issue 6458]).
+* Offline plugin management: We are introducing a new mechanism to manage plugins in an offline(air-gapped) 
+  environment. The previous approach would not completely download the plugins' dependency chain on the staging
+  machine. A new command called `prepare-offline-pack` will replace the existing `pack` and `unpack` subcommands 
+  in the `logstash-plugin` CLI to manage offline plugins. The `pack` and `unpack` subcommands have been 
+  deprecated ({lsissue}6393[Issue 6393]).
+* Fixed an issue where plugins couldn't be installed when HTTP proxy was enabled in Logstash 
+  deployments ({lsissue}6044[Issue 6044], {lsissue}5777[Issue 5777], {lsissue}5966[Issue 5966]).
+* Improved user facing error when `logstash.yml` is invalid ({lsissue}6509[Issue 6509]).
+* Fixed an issue where Logstash slowlog field `took_in_millis` reported values in microseconds instead of 
+  milliseconds ({lsissue}6476[Issue 6476]).
+* Added explicit `fsync` when writing to the checkpoint file to ensure date has been flushed to disk.
+* `jvm.options` is now loaded relative to the location of `logstash.yml` or if a custom `path.settings` is 
+  specified ({lsissue}6379[Issue 6379]).
+* Added ability to create a startup init script to the `system-install` utility ({lsissue}6148[Issue 6148]).
+
+[float]
+==== Input Plugins
+
+*`Kafka`*:
+
+* Made error reporting clearer when the connection to Kafka broker fails.
+* Made a change to set `kerberos` option only when using GSSAPI.
+
+*`Ganglia`*:
+
+* Store the name and value of ganglia metrics (https://github.com/logstash-plugins/logstash-input-ganglia/issues/2[Issue 2]).
+
+*`RabbitMQ`*
+
+* Fixed issue where metadata would be broken for strings values larger than 255 chars in message headers (https://github.com/logstash-plugins/logstash-input-rabbitmq/issues/94[Issue 94])
+
+==== Filter Plugins
+
+*`GeoIP`*:
+
+* `target` field should be merged and not completely overwritten by looked up GeoIP data (https://github.com/logstash-plugins/logstash-filter-geoip/issues/98[Issue 98]).
```

Comment:
Phrasing is a bit odd for the relnotes. Maybe say something like: "Fixed a bug that caused the `target` field to be overwritten by looked up GeoIP data. The `target` field is now merged instead of being completely overwritten."